templates/registration/logout.html

```diff
@@ -0,0 +1,20 @@
+{% extends "registration/registration_base.html" %}
+{% load i18n %}
+
+{% block title %}{% trans "Logged out" %}{% endblock %}
+
+{% block content %}
+<div class="container">
```

Comment:
I noticed that you are using the `container` class but previous PR's use `container-fluid`, im not an expert in bootstrap so do you know the difference?